CMakeLists.txt

```diff
@@ -12,13 +12,18 @@ target_link_libraries(spectre_v1 safeside)
 add_executable(spectre_v4 demos/spectre_v4.cc)
 target_link_libraries(spectre_v4 safeside)
 
+if (NOT ${CMAKE_SYSTEM_NAME} STREQUAL "Windows")
+  # Ret2Spec -- speculative execution using return stack buffers
+  set(CMAKE_CXX_FLAGS "-fomit-frame-pointer")
```

Comment:
I think this applies the flags to all subsequent targets. Try [`target_compile_options`](https://cmake.org/cmake/help/latest/command/target_compile_options.html) on the `ret2spec` target?

Reply:
Done.